examples/remoteBuild.md

```diff
@@ -1,5 +1,7 @@
 # remote targets
 
+##remote directories
```

Comment:
This isn't rendering correctly in markdown, try putting a space after the ##

Reply:
Thanks! Will note for future that GoLand and GitHub markdown render differently